astroid/raw_building.py

```diff
@@ -495,10 +505,17 @@ def _astroid_bootstrapping():
     )
     for _type in builtin_types:
         if _type.__name__ not in astroid_builtin:
-            cls = nodes.ClassDef(_type.__name__, _type.__doc__)
-            cls.parent = astroid_builtin
-            builder.object_build(cls, _type)
-            astroid_builtin[_type.__name__] = cls
+            klass = nodes.ClassDef(_type.__name__)
+            klass.parent = astroid_builtin
+            if _type.__doc__:
+                klass.postinit(
```

Comment:
This feels wrong. It would be better to always call `postinit` and assign a conditional and / or precomputed `doc_node` attribute.

Reply:
Fine with me!